kernel/src/main/java/org/kframework/kompile/KompileOptions.java

```diff
@@ -125,6 +125,9 @@ public boolean isKore() {
     @Parameter(names="--bison-stack-max-depth", description="Maximum size of bison parsing stack (default: 10000).")
     public long bisonStackMaxDepth = 10000;
 
+    @Parameter(names="--bison-parser-library", description="Generate a shared library rather than an executable for Bison parsers")
+    public boolean genBisonParserLibrary;
```

Comment:
Would it be possible to generate it by default as a shared library and have an executable that uses this library?
I'm only asking because it would eliminate the need for an extra option.
If it complicates the generation or other things, then this solution is fine.

Reply:
Yes, that's definitely possible and perhaps even desirable in the future. I'd need to benchmark it, and it would require a backend PR, so I'm inclined to leave it for now.